recipe/meta.yaml

```
@@ -100,26 +99,22 @@ outputs:
         - modin
         - modin.pandas
       commands:
-        # Skip on ppc64le because of distributed.comm.core.CommClosedError
-        - python -c "import modin.pandas as pd, modin.config as cfg; cfg.Engine.put('Dask'); df = pd.DataFrame([])"  # [not (linux and ppc64le)]
+        - python -c "import modin.pandas as pd, modin.config as cfg; cfg.Engine.put('Dask'); df = pd.DataFrame([])"
 
   - name: modin-ray
     build:
-      # There is no Ray in conda-forge for MacOS yet, see conda-forge/ray-packages-feedstock#2.
-      skip: true  # [py<38]
-      # ray is not yet compatible with Python 3.11
-      skip: true  # [not (linux64 or win64) or py>310]
+      # ray-default isn't available on s390x or python 3.12
+      skip: true  # [py<39 or py>311]
+      skip: true  # [s390x]
     requirements:
       host:
         - python
       run:
         - python
         - {{ pin_subpackage('modin-core', exact=True) }}
         - ray-default >=1.13.0
```

```suggestion
        - ray-default >=1.13.0,!=2.5.0
```
https://github.com/modin-project/modin/blob/0.26.1/setup.py#L11